dace/config_schema.yml

```diff
@@ -194,6 +194,14 @@ required:
                             explicit GPU block maps are not defined.
                         default: '32,1,1'
 
+                    dynamic_map_block_size:
+                        type: str
+                        title: Thread-Block size for GPU_ThreadBlock_Dynamic
+                        description: >
+                            Thread-Block size for maps using GPU_ThreadBlock_Dynamic
+                            scheduler.
+                        default: '128,1,1'
+
                     max_concurrent_streams:
```

Comment:
consider adding two booleans for the types of dynamic scheduler (both True by default)?